examples/js/postprocessing/EffectComposer.js

```diff
@@ -192,6 +192,22 @@ THREE.Pass = function () {
 
 };
 
+// Helper for passes that need a scene that simply has the whole viewport filled with a single quad.
+THREE.Pass.createFillQuadScene = function( material ) {
+
+	var fillQuad = {};
+
+	fillQuad.camera = new THREE.OrthographicCamera( - 1, 1, 1, - 1, 0, 1 );
+	fillQuad.scene = new THREE.Scene();
```

Comment:
I think we don't even need a scene anymore, I think this should work:

```js
renderer.render( quad, camera );
```

Reply:
Yep, it seems to, changed it now.